events/2022.09.Online/TD/Oracle/TDs/WoTWebThing+problemDetails+actionStatus.td.jsonld

```diff
@@ -0,0 +1,386 @@
+{
+	"@context": [
+		"https://www.w3.org/2019/wot/td/v1",
+		"https://www.w3.org/2022/wot/td/v1.1"
+	],
+	"@type": "Thing",
+	"title": "Blue Pump",
+	"description": "Blue Pump - Profile compliant WebThing (properties + actions + WebHook events)",
+	"id": "urn:com:blue:pump:data",
+	"profile": [
+		"https://www.w3.org/2022/wot/profile/baseline/v1",
+		"https://www.w3.org/2022/wot/profile/http-webhook/v1"
+	],
+	"version": {
+		"instance": "1.0.0"
+	},
+	"base": "http://w3cwotprofile.ddns.net:24042",
+	"securityDefinitions": {
+		"basic_sc": {
+			"scheme": "basic",
+			"in": "header"
+		}
+	},
+	"schemaDefinitions": {
+        "resultValue": {
+            "type": "object"
+        },
+ 		"problemDetails": {
+			"type": "string",
+			"title": "string",
+			"status": "number",
+			"detail": "string",
+			"instance": {
+				"$ref": "#/definitions/anyUri"
+			}
+		},
+        "actionStatus" : {
```

Comment:
this is not a json schema

Reply:
@egekorkan 
Not sure I understand. Why not? It passes your validation.

Reply:
I think what @egekorkan meant was that each schema should have a `type` term .. in the case of `actionStatus` most likely 
`"type": "object"`